### DISCUSSION TITLE
In an CI/CD environment how to skip installation of the hook?

### DISCUSSION THREAD

**Discussion by PreDer** [Cosium/git-code-format-maven-plugin#168] [Q&A] [answered]
So this plugin is designed to automatically install Git hooks for formatting code. However, in certain CI/CD environments like Microsoft Azure Pipelines, Git hooks might not be applicable or necessary. I would like to skip  the hook installation during the pipeline execution.

Do you provide a configuration option or parameter to skip the installation of Git hooks or got any suggest on that?

Thank you very much for this plugin and upfront for your answer!

**Comment by ashoktronix27** [comment, marked as answer]
Even I am looking for the same, `gcf.skip = true` Will help? or `<skip>${skipFormatCode}</skip>` >

**Reply by ashoktronix27** [reply]
@PreDer : 
This should solve the issue:                 <configuration>
                    <skip>${skipFormatCode}</skip>

Set in your jenkins pipeline maven goals.

**Comment by PreDer** [comment]
Setting -Dgcf.skip=true helped me in that case.

Thanks for the reference!
